spec/models/course_spec.rb

```diff
@@ -5,6 +5,21 @@
     before do
       @course = create(:course)
     end
+
+    context 'when a course is updated' do
+      context 'notify Kafka topic' do
+        let(:param) { rand }
+        subject(:responder) { CoursesResponder.new }
+
+        describe '#send_notification' do
+          it 'executes call method with self parameter' do
+            expect(@course).to receive(send_notification)
```

Comment:
Does this test pass? If it does then there is something wrong. The `send_notification` method should be called iff the section is changed, and it's not being changed here.